--- sodipodi-0.27/configure.in.wiget	Fri Nov 29 12:55:15 2002
+++ sodipodi-0.27/configure.in	Fri Nov 29 12:56:10 2002
@@ -62,6 +62,7 @@
 	fi
 )
 
+using_oaf="no"
 if $try_bonobo; then
 	AC_MSG_CHECKING(for Bonobo >= 0.37)
 	if $GNOME_CONFIG --libs bonobox_print > /dev/null 2>&1; then
@@ -95,13 +96,13 @@
 
 		AC_MSG_RESULT("$using_oaf")
 
-		AM_CONDITIONAL(USING_OAF, test x"using_oaf" = "xyes")
 	else    
 		AC_MSG_RESULT(not found)
 		SODIPODI_BONOBO_LIBS=
 		SODIPODI_BONOBO_CFLAGS=
 	fi
 fi
+AM_CONDITIONAL(USING_OAF, test x"using_oaf" = "xyes")
 
 AM_CONDITIONAL(BONOBO, $have_bonobo)
 AC_SUBST(SODIPODI_BONOBO_CFLAGS)
