diff -Nru sodipodi-0.20/Makefile.am sodipodi-0.20.new/Makefile.am
--- sodipodi-0.20/Makefile.am	Tue Oct 10 19:05:16 2000
+++ sodipodi-0.20.new/Makefile.am	Tue Oct 10 19:08:02 2000
@@ -23,14 +23,6 @@
 oafdir = $(datadir)/oaf
 oaf_DATA = sodipodi.oafinfo
 
-if BONOBO
-install-exec-local:
-	if test -e $(bindir)/sodipodi-bonobo; then \
-		rm -f $(bindir)/sodipodi-bonobo; \
-	fi ; \
-	ln -s $(bindir)/sodipodi $(bindir)/sodipodi-bonobo
-endif
-
 install-data-local:
 	@$(NORMAL_INSTALL)
 	if test -d $(srcdir)/glade; then \
diff -Nru sodipodi-0.20/Makefile.am~ sodipodi-0.20.new/Makefile.am~
--- sodipodi-0.20/Makefile.am~	Sun Sep 10 03:05:55 2000
+++ sodipodi-0.20.new/Makefile.am~	Tue Oct 10 19:05:16 2000
@@ -34,10 +34,10 @@
 install-data-local:
 	@$(NORMAL_INSTALL)
 	if test -d $(srcdir)/glade; then \
-	  $(mkinstalldirs) $(datadir)/sodipodi; \
+	  $(mkinstalldirs) $(DESTDIR)$(datadir)/sodipodi; \
 	  for uifile in $(srcdir)/glade/*; do \
 	    if test -f $$uifile; then \
-	      $(INSTALL_DATA) $$uifile $(datadir)/sodipodi; \
+	      $(INSTALL_DATA) $$uifile $(DESTDIR)$(datadir)/sodipodi; \
 	    fi \
 	  done \
 	fi
diff -Nru sodipodi-0.20/sodipodi.gnorba sodipodi-0.20.new/sodipodi.gnorba
--- sodipodi-0.20/sodipodi.gnorba	Mon Apr 10 09:39:27 2000
+++ sodipodi-0.20.new/sodipodi.gnorba	Tue Oct 10 19:06:59 2000
@@ -2,7 +2,7 @@
 type=exe
 repo_id=IDL:Bonobo/EmbeddableFactory:1.0 IDL:Bonobo/GenericFactory:1.0
 description=Sodipodi SVG Document factory
-location_info=sodipodi-bonobo
+location_info=sodipodi
 
 [embeddable:sodipodi-svg-doc]
 type=factory
@@ -14,7 +14,7 @@
 type=exe
 repo_id=IDL:Bonobo/EmbeddableFactory:1.0 IDL:Bonobo/GenericFactory:1.0
 description=Sodipodi SVG Document factory
-location_info=sodipodi-bonobo
+location_info=sodipodi
 
 [embeddable:canvas-item:sodipodi-svg-doc]
 type=factory
diff -Nru sodipodi-0.20/sodipodi.oafinfo sodipodi-0.20.new/sodipodi.oafinfo
--- sodipodi-0.20/sodipodi.oafinfo	Tue May  9 01:03:36 2000
+++ sodipodi-0.20.new/sodipodi.oafinfo	Tue Oct 10 19:07:13 2000
@@ -2,7 +2,7 @@
 
 <oaf_server iid="OAFIID:embeddable-factory:sodipodi-svg-doc:4fda0c32-8996-4196-ac88-a79b5888e269"
             type="exe"
-            location="sodipodi-bonobo">
+            location="sodipodi">
 
 	<oaf_attribute name="repo_ids" type="stringv">
 		<item value="IDL:Bonobo/EmbeddableFactory:1.0"/>
@@ -30,7 +30,7 @@
 
 <oaf_server iid="OAFIID:embeddable-factory:canvas-item:sodipodi-svg-doc:60761039-5a0e-4da0-848e-86d406399b8c"
             type="exe"
-            location="sodipodi-bonobo">
+            location="sodipodi">
 
 	<oaf_attribute name="repo_ids" type="stringv">
 		<item value="IDL:Bonobo/EmbeddableFactory:1.0"/>
