diff -urN aaa/sodipodi-0.33/sodipodi.desktop.in sodipodi-0.33/sodipodi.desktop.in
--- aaa/sodipodi-0.33/sodipodi.desktop.in	2003-05-08 20:51:48.000000000 +0200
+++ sodipodi-0.33/sodipodi.desktop.in	2003-12-13 00:28:16.000000000 +0100
@@ -6,7 +6,7 @@
 Version=1.0
 
 Type=Application
-Categories=Application;Graphics;
+Categories=GTK;GNOME;Graphics;VectorGraphics;
 MimeType=image/svg+xml
 
 FilePattern=sodipodi
