--- sodipodi-0.24.1/src/display/Makefile.am~	Mon Nov 26 02:26:46 2001
+++ sodipodi-0.24.1/src/display/Makefile.am	Mon Nov 26 02:26:46 2001
@@ -37,7 +37,7 @@
 
 INCLUDES = \
 	-I$(includedir) \
-	$(GNOME_INCLUDEDIR)
+	$(SODIPODI_CFLAGS)
 
 noinst_LIBRARIES = libspdisplay.a
 
--- sodipodi-0.24.1/src/bonobo/Makefile.am~	Mon Nov 26 02:37:54 2001
+++ sodipodi-0.24.1/src/bonobo/Makefile.am	Mon Nov 26 02:37:54 2001
@@ -5,7 +5,8 @@
 
 INCLUDES = \
 	-I$(includedir) \
-	-I$(SODIPODI_CFLAGS)
+	-I$(SODIPODI_CFLAGS) \
+	-I$(SODIPODI_BONOBO_CFLAGS)
 
 noinst_LIBRARIES = libspbonobo.a
 
